lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -38,27 +40,48 @@ Stream<Patch> intlImporter(
 
   if (!needsIntlImport) return;
 
-  final intlUri = 'package:${projectName}/src/intl/${projectName}_intl.dart';
+  final currentFilePath = Platform.script.toFilePath();
+  final currentDirectory = path.dirname(currentFilePath);
+
+  final intlUri = 'package:$projectName/src/intl/${projectName}_intl.dart';
+  final relativePathToIntlDir = path.relative(currentDirectory, from: intlUri);
```

Comment:
Does this (computing a relative path between a URI and a file system path) work in general? I think you could do this more simply as
`intlDirectory = path.join(context.root, 'lib/src/intl/${projectName}_intl.dart');`
`path.relative(intlDirectory, path.current)`
and then you don't need to manually compute relativeImportPath.

Reply:
Updated as you suggested.

Reply:
> Does this (computing a relative path between a URI and a file system path) work in general? I think you could do this more simply as `intlDirectory = path.join(context.root, 'lib/src/intl/${projectName}_intl.dart');` `path.relative(intlDirectory, path.current)` and then you don't need to manually compute relativeImportPath.

I have a one-line `final currentDirectory = Directory.current`; before   `final intlDirectory = path.join(currentDirectory.path, 'lib/src/intl/${projectName}_intl.dart');`